src/ontology/edits/oeo-physical.omn

```diff
@@ -2593,6 +2669,9 @@ Class: Sea
         GeographicRegion
     
     
+Class: Sector
```

Comment:
Here a class `Sector` pops up without any definition or any relation to other classes. I think this is caused by changing `CarbonDioxideEmissionTrafficSector` to subclass of `Sector`. The class `Sector` is defined in the oeo-social. Does the import work not properly?

Reply:
oh yes I moved it manually now from physical to social